documentation/modules/rn-2.2.adoc

```diff
@@ -9,21 +9,33 @@ You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {vir
 
 The release notes describe new features and enhancements, known issues, and technical changes.
 
-// [id="technical-changes-22_{context}"]
-// == Technical changes
-//
-// .TBD
-//
-// TBD
-//
-// [id="new-features-and-enhancements-22_{context}"]
-// == New features and enhancements
-//
-// This release adds the following features and improvements.
-//
-// .TBD
-//
-// TBD
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+.Setting interval between precopies
+
+You can set the precopy interval.
+
+.Handling RHV VMs with USB enabled
+
+You can migrate an RHV VM with a host device of the USB type mapped to it, although the mapping is not supported on the target side.
```

Comment:
What does this mean? "mapping is not supported on the target side" suggests that the mapping for migration does not work. I suspect that what it means is that the user must add a mapping for the host USB device (**not** host device of the USB type) manually after migration. Is this what you mean? We do not refer to "target side" in the docs, so I presume you mean in the target environment or on the target cluster. Is this the one that you plan to move to Known Issues? If so, it needs a BZ link.

Reply:
@apinnick 
No, this is a technical change. But the wrong warning message that accompanies the migration is a known issue. https://bugzilla.redhat.com/show_bug.cgi?id=2025989

Reply:
Yes, the bug does refer to USB enablement, so you need to remove the part about the host device. It refers only to the setting.

Reply:
OK